pkg/controller/podgc/gc_controller_test.go

```diff
@@ -57,10 +57,7 @@ func compareStringSetToList(set sets.String, list []string) bool {
 			return false
 		}
 	}
-	if len(list) != len(set) {
```

Comment:
Maybe move this to the head of the function, it's more effective.

Reply:
Exactly. Thx.